### DISCUSSION TITLE
Separate QuantUMS for groups of samples

### DISCUSSION THREAD

**Discussion by pisistrato** [vdemichev/DiaNN#1313] [Q&A] [answered]
Hi Vadim,

Correct me if I am wrong, but it is my understanding that the QuantUMS algorithm, like the maxLFQ one, assumes sample homogeneity, i.e., the majority/large portion of the proteome does not change across samples.
What would be the best strategy to process a dataset consisting of two distinct groups of samples? E.g. full proteome and organelle proteome, or full proteome and secretome?
I would like to process the entire dataset at once so that protein grouping is the same.
MaxQuant, for instance, allows you to run the LFQ algorithm independently for each defined sample group. Is this possible in DIA-NN?

Thanks
-Piero

**Comment by vdemichev** [comment, marked as answer]
Hi Piero,

'most proteins don't change' is the assumption for normalisation, this is not necessary for QuantUMS or MaxLFQ.

> I would like to process the entire dataset at once so that protein grouping is the same.

Can create a DIA-based lib from the entire dataset (with DIA-NN 1.9.1 or later), then analyse each type of samples separately (with protein inference disabled) - this is best for normalisation to work optimally. Protein groups will be inferred during the creation of the DIA lib.

Best,
Vadim

**Comment by pisistrato** [comment]
Hi @vdemichev , sorry to go back to this, just to clarify, I get a warning about using MBR when I re-analyze each subset individually. since the FDR will be different for each dataset, I assume I should have it enabled right?

**Reply by pisistrato** [reply]
Nevermind, I am sorry I have overlooked the docs, there it is specified what to do :)

> Always use MBR for quantitative analyses (as opposed to analyses just aimed at creating an empirical library), unless you are analysing with a DIA-based empirical library (in that case do not use MBR)
